ionic-app/pages/_app.js

```diff
@@ -0,0 +1,31 @@
+import Head from 'next/head';
+import Script from 'next/script';
+
+import 'tailwindcss/tailwind.css';
+import '@ionic/react/css/core.css';
+import '@ionic/react/css/padding.css';
+import '@ionic/react/css/float-elements.css';
+import '@ionic/react/css/text-alignment.css';
+import '@ionic/react/css/text-transformation.css';
+import '@ionic/react/css/flex-utils.css';
+import '@ionic/react/css/display.css';
+
+import '../styles/global.css';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
import "../styles/global.css";
```